apps/etl-func/src/adapters/redis/config.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const redisConfigSchema = z.object({
+  accessKey: z.string().min(1),
+  pingInterval: z.number(),
```

Comment:
Can we set a min here? I would suggest 1 second here
```suggestion
  pingInterval: z.number().min(1000),
```